### ISSUE TITLE
Bad performance with clipboard=unnamedplus while having a remote workspace open

### ISSUE THREAD

**Issue by SHanded** [asvetliakov/vscode-neovim#555]
### **Environment:**
VSCode + Neovim latest for Windows + vscode-neovim Plugin

### **Issue:**
Deleting characters in any file (even new file) while a workspace with a remote folder is open takes longer than expected (1-2 sec).
Yanking lines also takes 1-2 sec.

### **Steps to reproduce:**

Setting clipboard=unnamedplus must be active.

- Open a workspace that has a remote folder included (e.g. an SMB share over VPN).
- Open new Untitled file (<kbd>Ctrl+N</kbd>).
- Type a few lines of text.
- Go to a line and press <kbd>dd</kbd>.
- It will take 1-2 sec for the line to disappear.

### **In contrast:**

- Open a workspace that has a remote folder included (e.g. an SMB share over VPN).
- Open new Untitled file (Ctrl+N).
- Type a few lines of text.
- Go to a line and press <kbd>"_dd</kbd>. (black hole register)
- Line disappears instantly.

### **In contrast 2:**

- Close workspace.
- Open new Untitled file.
- Type a few lines of text.
- Go to a line and press <kbd>dd</kbd>.
- Line disappears instantly.

**Comment by Shatur95** [comment]
Probably caused by your plugins. I do not have this issue.

**Comment by SHanded** [comment]
Nope, even with just vscode-neovim this occurs. My guess is there's some sort of issue with win32yank.exe in combination with remote files going on.

Did you replicate my environment to test this out?

**Comment by Shatur95** [comment]
> Nope, even with just vscode-neovim this occurs.

No, I meant neovim plugins. Have you tried with a clean config (except `clipboard=unnamedplus`)?

> Did you replicate my environment to test this out?

Yes, it works on my Windows machine quite fast.

**Comment by SHanded** [comment]
Oh yeah, I don't have any neovim plugins, my config is only this atm:

```
:set ignorecase
:set smartcase
:set clipboard=unnamedplus
```

**Comment by Shatur95** [comment]
I have all these options enabled too... What version of Windows do you have? Are you using WSL?

**Comment by SHanded** [comment]
Win10 x64, not using WSL

**Comment by Shatur95** [comment]
What `:checkhealth` show you (in Neovim)?
